schemas/json/widget/widget.schema.v1.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$id": "https://altinncdn.no/schemas/json/widget/widget.schema.v1.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
I guess it's time to start using the latest draft for our brand new stuff? `https` as bonus :)

`https://json-schema.org/draft/2019-09/schema`

http://json-schema.org/draft/2019-09/release-notes.html